sql_patch.php

```diff
@@ -16,7 +16,20 @@
 $ignoreAuth = true; // no login required
 
 require_once('interface/globals.php');
+require_once('library/sql.inc');
 require_once('library/sql_upgrade_fx.php');
+require_once('version.php');
```

Comment:
TODO
Will remove the above require for sql.inc and version.php (these are already brought in by globals.php).